examples/runtimes/java_eq_fun.tf

```diff
@@ -0,0 +1,31 @@
+# Build the zip archive whenever the filename changes.
```

Comment:
What's this file intended to do?

Reply:
I will remove it, but it was the way I built the Java package using Terraform since I lacked Gradle knowledge.

I eventually found the way to do it better using Gradle - https://github.com/terraform-aws-modules/terraform-aws-lambda/pull/612/files#diff-7706d0318e25b3172ede117c34b5ba0428d12d694366670572f1a0843c2d7d84R30-R40